app/src/main/java/org/desarrolladorslp/teckersapp/ui/messages/CircleTransform.kt

```diff
@@ -0,0 +1,49 @@
+package org.desarrolladorslp.teckersapp.ui.messages
+
+import android.graphics.Bitmap
+import android.graphics.BitmapShader
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.RectF
+import android.graphics.Shader
+
+import com.squareup.picasso.Transformation
+
+class CircleTransform
+    (
+    private val radius: Int, private val margin: Int // dp
```

Comment:
what `//dp` means?

Reply:
its a measure for the margins but i take it out.